test/CappedCrowdsale.js

```diff
@@ -62,7 +70,7 @@ contract('CappedCrowdsale', function ([_, wallet]) {
   describe('ending', function () {
 
     beforeEach(async function () {
-      await advanceToBlock(this.startBlock - 1)
+      await increaseTime(moment.duration(1, 'week'))
```

Comment:
Okay so the problem is that we don't know how testrpc gets its block timestamps. I don't see how the current usage of `increaseTime` avoids the problem. We still run the risk of it increasing time by a bit too much, which is afaict the same risk we'd run with the alternative. Do we agree on that?

As for point 1, have you noticed that the `increaseTime` helper already calls both `evm_increaseTime` and `evm_mine`?

I still think we should define an `increaseTimeTo` helper, but if you don't agree that it's a good idea, can you at least define variables `timeToStart`, `duration`, etc., and use those instead of hardcoding their values?